tools/upgrade_versions.py

```diff
@@ -114,13 +114,17 @@ def update(self, user_name, user_email):
 
     def _bump_release(self, pkg, log=None, user_name=None, user_email=None):
```

Comment:
Using a list isn't useful for our current usage, I'm applying your suggestion and renaming the variable to something better.